.github/workflows/send-redirects-to-staging.yml

```diff
@@ -1,11 +1,14 @@
 # Terraform Action to lint and apply updated redirects
 name: Publish Staging Redirects
 on:
-  workflow_dispatch:
-    inputs:
-      branch:
-        description: 'Branch with draft redirects'
-        required: true
+  pull_request:
```

Comment:
Do we have to specify all of the different types?

Reply:
Nop, when a Github Actions workflow is configured to run on pull requests, by default it will be triggered on three different types of activities: opened, synchronize, and reopened so removing them